apps/daimo-mobile/src/i18n/languages/en.ts

```diff
@@ -0,0 +1,783 @@
+import { LanguageDefinition } from "../generate/languageDefinition";
+
+export const en: LanguageDefinition = {
```

Comment:
@Ethanol48 Added a type LanguageDefinition -- should should you errors if you are missing anything